### DISCUSSION TITLE
Ext ideas anyone?

### DISCUSSION THREAD

**Discussion by nav-github01001** [EpikCord/EpikCord.py#62] [Q&A] [answered]
I am thinking of an extension, what kind of ideas for it you have? @TheUntraceable @Jerry-py @itsnotrin

**Comment by Jerry-py** [comment, marked as answer]
Ext pages?

**Reply by nav-github01001** [reply]
that @TheUntraceable will implement

**Reply by nav-github01001** [reply]
any other ideas

**Reply by TheUntraceable** [reply]
Have this implemented, sorta.

**Comment by TheUntraceable** [comment]
You *could* work on message commands if you really wanted.

**Comment by LisABC** [comment]
Voice receive stuff ? Maybe do that later when you are done with fully implementing everything first
